packages/nova-react/src/graphql/hooks.ts

```diff
@@ -10,6 +10,7 @@ import type {
   RefetchFn,
   FetchPolicy
 } from "./types";
+import type { Disposable } from 'relay-runtime';
```

Comment:
I see why you have the peerDependency now. 

Is there a way we can use/create a compatible type? That way we do not need to take the whole dependency for just one type.

Reply:
From looking at the types in here [@types/relay-runtime](https://www.npmjs.com/package/@types/relay-runtime?activeTab=code) (Can't direct link, but they're located at `/@types/relay-runtime/lib/util/RelayRuntimeTypes.d.ts`) the types to support this seem really simple :) 

```ts
/**
 * Represents any resource that must be explicitly disposed of. The most common
 * use-case is as a return value for subscriptions, where calling `dispose()`
 * would cancel the subscription.
 */
export interface Disposable {
    dispose: DisposeFn;
}

export type DisposeFn = () => void;
```
```

Reply:
Thanks for the suggestion! Code is updated :)